Makefile

```diff
@@ -94,37 +94,37 @@ benchmark:
 
 # Not used in CI, tests are split in pkg, tempodb, tempodb-wal and others in CI jobs
 .PHONY: test-with-cover
-test-with-cover: test-serverless
+test-with-cover: tools test-serverless
```

Comment:
Alternative to the make target dependency here, we could add `make tools` in the CI as a step.